app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -835,7 +835,7 @@ class BrowserTabViewModel @Inject constructor(
         setAdClickActiveTabData(url)
 
         // we expect refreshCta to be called when a site is fully loaded if browsingShowing -trackers data available-.
-        if (!currentBrowserViewState().browserShowing) {
+        if (!currentBrowserViewState().browserShowing && !currentBrowserViewState().maliciousSiteDetected) {
```

Comment:
@malmstein @nalcalag pinging you here since you have more context on the omnibar and onboarding, in case you see any issues with this

Reply:
Thanks for the ping. This looks good and the proper approach to deal with this at this point.